src/fides/api/ops/service/connectors/fides_connector.py

```diff
@@ -142,17 +151,17 @@ def close(self) -> None:
 
 def filter_fides_connector_datasets(
     connector_configs: List[ConnectionConfig],
-) -> List[Tuple[str, ConnectionConfig]]:
+) -> Set[str]:
```

Comment:
i'm actually not sure on the cardinality between connection configs and datasets - it may be "many to one" - in which case this would be a _little_ helpful. mainly just figured that we care about distinct datasets, so made it a set to make that intention clear. maybe that's not a very pythonic thing to do though!

Reply:
A set it totally fine from a Python prospective. I was just wondering if it would ever be possible. Sounds like there is at least the potential so this makes sense.